async_rediscache/types/queue.py

```diff
@@ -260,7 +244,7 @@ class RedisTask:
     task as done as long as the owner queue is still alive.
     """
 
-    def __init__(self, value: RedisValueType, owner: RedisQueue) -> None:
+    def __init__(self, value: RedisValueType, owner: RedisTaskQueue) -> None:
```

Comment:
I don't see `RedisQueue` renamed as `RedisTaskQueue` in the diff?

Reply:
RedisTaskQueue is a pre-existing subclass of RedisQueue. Going off the signature in one of the getters below, it seems there is a mismatch between expected in class, and return. This left us with two options, return a RedisQueue, or change the init as I've done. The reason I believe this is preferable over changing the return is that the TaskQueue class is more useful than a regular queue and is no harder to create. It also makes more logical sense with the name of this class.

P.S the reason this change was included here was because it's a breaking change. It's a very small change, so it made more sense to Kaizen it than make two consecutive breaking releases.